etc/config-template.toml

```diff
@@ -214,7 +214,7 @@
 ## When Region size change exceeds this config, TiKV will check whether the Region should be split
 ## or not. To reduce the cost of scanning data in the checking process, you can set the value to
 ## 32MB during checking and set it back to the default value in normal operations.
-# region-split-check-diff = "6MB"
+# region-split-check-size-diff = "6MB"
```

Comment:
For compatible issue, we should keep the origin config and noted as deprecated.

Reply:
@Connor1996  I think it's OK because of the `region-split-check-diff` has been deleted, the config will be checked by `tikv check-config` before upgrading.

Reply:
@lonng And the check will fail and need to resolve by hand?